.circleci/config.yml

```diff
@@ -203,8 +203,8 @@ jobs:
             export PATH="$HOME/miniconda3/envs/hydra/bin:$PATH"
             export NOX_PYTHON_VERSIONS=<< parameters.py_version >>
             export PLUGINS=<< parameters.test_plugin >>
-            pip install nox
-            pip install dataclasses
+            pip install nox --progress-bar off
+            pip install dataclasses --progress-bar off
```

Comment:
can you check if this dataclasses is actually needed? I don't know what it's doing here (any anywhere else).

Reply:
`dataclasses` is needed for python 3.6 
here is one of the failure runs https://app.circleci.com/pipelines/github/jieru-hu/hydra/836/workflows/f30999e7-1a1c-414f-ba7f-4d56b059e915/jobs/8566

the stacktrace
```
Installing collected packages: appdirs, zipp, importlib-metadata, filelock, importlib-resources, distlib, six, virtualenv, colorlog, argcomplete, py, nox
Successfully installed appdirs-1.4.4 argcomplete-1.12.1 colorlog-4.4.0 distlib-0.3.1 filelock-3.0.12 importlib-metadata-2.0.0 importlib-resources-3.2.0 nox-2020.8.22 py-1.9.0 six-1.15.0 virtualenv-20.1.0 zipp-3.4.0
Traceback (most recent call last):
  File "/home/circleci/miniconda3/envs/hydra/bin/nox", line 8, in <module>
    sys.exit(main())
  File "/home/circleci/miniconda3/envs/hydra/lib/python3.6/site-packages/nox/__main__.py", line 61, in main
    tasks.final_reduce,
  File "/home/circleci/miniconda3/envs/hydra/lib/python3.6/site-packages/nox/workflow.py", line 49, in execute
    return_value = function_(*args, global_config=global_config)
  File "/home/circleci/miniconda3/envs/hydra/lib/python3.6/site-packages/nox/tasks.py", line 60, in load_nox_module
    "user_nox_module", global_config.noxfile
  File "<frozen importlib._bootstrap_external>", line 399, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 823, in load_module
  File "<frozen importlib._bootstrap_external>", line 682, in load_module
  File "<frozen importlib._bootstrap>", line 265, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/circleci/project/noxfile.py", line 6, in <module>
    from dataclasses import dataclass
ModuleNotFoundError: No module named 'dataclasses'

```